build/test-page/index.html

```diff
@@ -7,7 +7,7 @@
         <script type = 'text/javascript' src = 'https://d3js.org/d3.v3.min.js'></script>
         <script type = 'text/javascript' src = 'https://rawgit.com/SheetJS/js-xlsx/master/dist/xlsx.full.min.js'></script>
         <script type = 'text/javascript' src = 'https://rawgit.com/RhoInc/Webcharts/master/build/webcharts.js'></script>
-        <script type = 'text/javascript' src = 'https://rawgit.com/RhoInc/rave-xplorer/master/build/raveXplorer.js'></script>
+        <script type = 'text/javascript' src = 'https://rawgit.com/RhoInc/crf-heat-map/master/build/crf-heat-map.js'></script>
```

Comment:
`crf-heat-map.js` should be `crfHeatMap.js` here?

Reply:
good call, thank ya